src/components/wishList/WishListView.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import AlbumGrid from '../AlbumGrid';
+import axiosInstance from '../../apis/axiosClient';
+
+const WishListView = () => {
+  const [albums, setAlbums] = useState();
+  const [wishListId, setWishListId] = useState();
+
+  //fetch wishlist album from API
+  useEffect(() => {
+    const fetchWishlist = async () => {
+      try {
+        const response = await axiosInstance.post(`/wishlist/`); //use axiosInstance to send cookie token with request
+        const wishlistData = response.data.wishlist;
+        setWishListId(wishlistData._id); //set the state for wishlist id
+        setAlbums(wishlistData.albums); //set the state for wishlist albums
+        // Store each album id from wishlist in local storage
+        wishlistData.albums.forEach((album) => {
+          localStorage.setItem(
+            `wishListAlbums-${album._id}`,
+            JSON.stringify(true)
+          );
+        });
+      } catch (error) {
+        console.error('Error fetching wishlist:', error);
+      }
+    };
+
+    fetchWishlist();
+  }, []);
+
+  return <AlbumGrid albums={albums} wishListId={wishListId} />;
```

Comment:
![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/ec01cbfd-62ed-4bda-83f1-805543d9be86)

Currently our wishlist page looks relatively blank; may be worthwhile to add a container to wrap the `AlbumGrid` and maybe a header to indicate that this is the wishlist page.

Reply:
You may also want to have some sort of empty state message when the user's wishlist has no albums on it.

Reply:
Done